src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -102,22 +101,21 @@ export class CdpTarget {
    */
   async #unblock() {
     try {
+      NetworkProcessor.create(this.cdpClient, this.#eventManager);
```

Comment:
This seems out of place now.

Reply:
Ideally it should be moved to CommandProcessor, however once that is done all network tests start to fail.

Reply:
I think the class name is misleading we should call it NetworkManager. Similar example is the logManager, where we have no Processor, as there are no commands.

Reply:
Done